opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -103,6 +103,28 @@ def _transcribe_property(prop):
         return str(prop)
 
 
+def _extract_timecode(mob):
+    """Given a mob with a single timecode slot, return the timecode in that 
+    slot or None if no timecode slots could be found.
+    """
+
+    tc_list = [
+        s.segment['Start'].value for s in mob.slots()
+        if s.segment.media_kind == 'Timecode'
+    ]
+
+    if len(tc_list) == 1:
+        return tc_list[0]
+    elif len(tc_list) > 1:
+        raise otio.exceptions.NotSupportedError(
+            "Error: mob has more than one timecode track, this is not"
```

Comment:
I think we should call it timecode slot instead of track to stay in the AAF terminology. 'track' in the context of AAF confuses me :)